### PR TITLE
Block/headings

### PR DESCRIPTION
**Pull request by Baskov-Aleksey-Viktorovich**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -50,7 +50,67 @@
         </div>
     </nav>
 <!-- SubHeader -->
-
+<!--  Header -->
+<header class="header">
+    <div class="header__container">
```
Comment (Anatolych97) [inline index.html:55]: Use BEM, Element can't contain another element
header-container  - it's block
header__container - it's element
header-container--dense - it's modifier, can be used with elements and blocks

File: index.html [hunk 2]
```diff
@@ -50,7 +50,67 @@
...
+        <nav class="header__menu">
+            <a href="#">
```
Comment (Anatolych97) [inline index.html:57]: Logo should lead to root website URL

File: index.html [hunk 3]
```diff
@@ -50,7 +50,67 @@
...
+                <img src="/src/img/icons/logotipe.png" alt="logo">
+            </a>
+            <div class="menu">
```
Comment (Anatolych97) [inline index.html:60]: Wrong intent  or mistake?

File: index.html [hunk 4]
```diff
@@ -50,7 +50,67 @@
...
+                <li class="menu__item"><a href="/">latest</a></li>
+                <li class="menu__item"><a href="/">featured</a></li>
+                <li class="menu__item"><a href="/">about</a></li>
+                <li class="menu__item"><a href="/">blog</a></li>
+                <li class="menu__item"><a href="/">Contact</a></li>
+            </ul>
+        </div>
+            <div class="header__search"></div>
+           <div class="header__truck">
+               <img src="#" alt="#">
```
Comment (Anatolych97) [inline index.html:73]: Useless

File: index.html [hunk 5]
```diff
@@ -50,7 +50,67 @@
...
+            <div class="headings__container">
+                <div class="headings__wraper">
+                    <div class="headings__bold">
+                        <div class="headings__title">Bold Headings</div>
+                            <h1>Heading level 1</h1>
+                            <h2>Heading level 2</h2>
+                            <h3>Heading level 3</h3>
+                            <h4>Heading level 4</h4>
+                            <h5>Heading level 5</h5>
+                            <h6>Heading level 6</h6>
```
Comment (Anatolych97) [inline index.html:91]: Where are elements classes?

**Comment by Anatolych97** [comment]
I don't styling file. 
Pay attention each block should have own styling file

**Comment by Anatolych97** [comment]
Also don't create specifity in css. Use & to generate child selectors without growing specifity

File: index.html [hunk 6]
```diff
@@ -50,7 +50,67 @@
...
+                    </div>
+                    <div class="headings__size">
+                        <p class="item-heading">42 pt</p>
```
Comment (Anatolych97) [inline index.html:94]: In my opinion there will be better to use span

File: index.html (hunk 3 above)
Comment (Baskov-Aleksey-Viktorovich) [reply index.html:60]: mistake(

File: index.html (hunk 5 above)
Comment (Baskov-Aleksey-Viktorovich) [reply index.html:91]: +

File: index.html (hunk 6 above)
Comment (Baskov-Aleksey-Viktorovich) [reply index.html:94]: +

**Pull request closed without merging**